src/pages/topics/[slug].tsx

```diff
@@ -25,6 +36,45 @@ const Topic: NextPage<TopicProps> = ({ name, description, ctas }) => {
                 description={description}
                 ctas={ctas}
             />
+            <div
+                style={{
+                    display: 'flex',
+                    gap: '20px',
+                    flexWrap: 'wrap',
+                }}
+            >
+                {contentCategories.map(c =>
+                    getL1Content()
+                        .filter(a => a.pillCategory == c)
+                        .map(d => (
```

Comment:
Same here about being more explicit with the naming of arguments/parameters `d` in fact we can even use destructuring here like so `map(d => ({title, pillCategory, thumbnail, description, contentDate, title})`